nixos/modules/hardware/hid-fanatecff.nix

```
@@ -0,0 +1,34 @@
+{ pkgs, lib, config, ... }:
+
+with lib;
+
+let
+  cfg = config.hardware.hid-fanatecff;
+  kernelPackages = config.boot.kernelPackages;
+in {
+  options.hardware.hid-fanatecff = {
+    enable = mkOption {
+      type = types.bool;
+      default = false;
+      example = true;
+      description = lib.mdDoc ''
```

```suggestion
      description = ''
```

`lib.mdDoc` is deprecated and options descriptions are now in Markdown by default.
See https://github.com/NixOS/nixpkgs/blob/master/lib/options.nix#L407